pkg/apis/apps/v1beta1/conversion.go

```diff
@@ -86,6 +86,10 @@ func Convert_v1beta1_StatefulSetSpec_To_apps_StatefulSetSpec(in *appsv1beta1.Sta
 	if err := autoConvert_v1beta1_StatefulSetSpec_To_apps_StatefulSetSpec(in, out, s); err != nil {
 		return err
 	}
+	if in.PersistentVolumeClaimDeletePolicy != nil {
```

Comment:
isn't this already handled by autoConvert?

Reply:
yes, you seem to be correct. At least the tests pass...